<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>584</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spout / Drain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item row="1" column="0">
     <widget class="SinkComponent" name="componentSpout" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="SinkComponent" name="componentDrain" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelQueue">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>29</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_localhost"/>
    <addaction name="actionOpen_URL"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Spout">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Queue</string>
    </property>
    <addaction name="action_Choose_existing_queue"/>
    <addaction name="action_Declare_new_queue"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Spout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Choose_existing_queue"/>
   <addaction name="action_Declare_new_queue"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_localhost">
   <property name="text">
    <string>Open &amp;localhost</string>
   </property>
  </action>
  <action name="actionOpen_URL">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Choose_existing_queue">
   <property name="text">
    <string>&amp;Choose existing queue</string>
   </property>
  </action>
  <action name="action_Declare_new_queue">
   <property name="text">
    <string>&amp;Declare a new queue</string>
   </property>
  </action>
  <action name="action_Start_sending_messages">
   <property name="text">
    <string>&amp;Start sending messages</string>
   </property>
  </action>
  <action name="action_Start_removing_messages">
   <property name="text">
    <string>&amp;Start removing messages</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SinkComponent</class>
   <extends>QWidget</extends>
   <header location="global">sinkcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
